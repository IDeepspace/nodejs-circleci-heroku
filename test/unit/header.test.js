import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/components/header/index';

test('Link changes the class when hovered', () => {
  const component = renderer.create(<Header> Facebook</Header>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
