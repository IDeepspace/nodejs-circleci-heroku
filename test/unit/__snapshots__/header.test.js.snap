// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <div>
    CBS Lite
  </div>
  <div>
    <img
      alt="avatar"
      src="test-file-stub"
    />
    <div>
      xinchen
    </div>
  </div>
</div>
`;
